docs/manual/lore/history.md

```diff
@@ -14,6 +14,7 @@ Notable Events:
 1. **Dvaered Revolts** (UST 307)
 1. **Sorom Plague** (UST 328)
 1. **Creation of House Dvaered** (UST 331)
+1. **Quarantine of Sorom** (UST 333)
```

Comment:
I understand this is the start of the quarantine, right? Maybe it would make sense for it to be earlier.

Reply:
I just wanted some space b/w Dvaered formation and the Quarantine; feel free to move it earlier so long as it remains after the DV formation, because the document will have to be changed considerably otherwise.

Reply:
Mmmm, I guess it would make sense if the Empire was too busy with House Dvaered, so they would not really focus on the plague and officially lock it down. I'll mark this as resolved and think about it a bit more.

Reply:
The Proteron doc, I mean